scrnaseq.Rmd

```diff
@@ -585,8 +585,8 @@ sc = purrr::map(list_names(sc), function(n) {
   genes_g2m_exists = genes_g2m[,2] %in% rownames(sc[[n]])
   if (sum(genes_s_exists)>=10 & sum(genes_s_exists)>=10){
     sc[[n]] = Seurat::CellCycleScoring(sc[[n]], s.features=genes_s[genes_s_exists,2], g2m.features=genes_g2m[genes_g2m_exists,2], set.ident=FALSE, verbose=FALSE)
+    sc[[n]][["CC.Difference"]] = sc[[n]][["S.Score", drop=TRUE]] - sc[[n]][["G2M.Score", drop=TRUE]]
```

Comment:
Is the "drop=TRUE" needed here? Didn't it work before?

Reply:
Well if you for example do sc[["S.Score"]] it will return a dataframe. If you include drop it will return a vector. Like in basic R but vice versa?